<launch>
    <node pkg="gp_wind_regression" type="gp_wind_regression" name="gp_wind_regression_z" output="screen" />

	<param name="switch_xyz" type="int" value="2" /> <!-- Paramter to switch the regression between x,y,z. x=0, y=1, z=2.-->
	<param name="prediction_init_pub_topic" type="string" value="/gp_disturb_reg/predInit/z" /> <!-- Parameter to specfy topic name for for predInit.-->
	<param name="mean_pub_topic" type="string" value="/gp_disturb_reg/mu/z" /> <!-- Parameter to specfy topic name for mean.-->
	<param name="variance_pub_topic" type="string" value="/gp_disturb_reg/var/z" /> <!-- Parameter to specfy topic name for variance.-->
	<param name="mean_p_2std_dev_pub_topic" type="string" value="/gp_disturb_reg/mu_p_2std_dev/z" /> <!-- Parameter to specfy topic name for mean_p_2std_dev_pub_topic.-->
	<param name="mean_m_2std_dev_pub_topic" type="string" value="/gp_disturb_reg/mu_m_2std_dev/z" /> <!-- Parameter to specfy topic name for mean_m_2std_dev_pub_topic.-->
        <param name="feature_data_sub_topic" type="string" value="/dev/gp/feature_z_filtered" /> <!-- Parameter to specfy topic name for GP feature data. -->
	<param name="file_gp_initial" type="string" value="gp_initial_z.dat" /> <!-- Parameter to specfy file name for gp_initial.-->
	<param name="create_new_gp_initial" type="bool" value="true" /> <!-- Paramter to specify whether to create new gp_initial.-->
	<param name="use_gp_initial" type="bool" value="true" /> <!-- Paramter to use LSTM feature by utilizing gp_initial.-->
	<param name="use_sparse_gp_initial" type="bool" value="false" /> <!-- Paramter to use sparse_gp_initial.-->
	<param name="sizefactor_sparse_gp_initial" type="double" value="0.7" /> <!-- Paramter to set sizefactor for sparse_gp_initial.-->
       <param name="lambda" type="double" value="0.5" /> <!-- Paramter to set sizefactor for sparse_gp_initial.-->
	<!-- Load commmon parameters. -->
	<include file="$(find gp_wind_regression)/launch/common_params.xml" />

</launch>
